Docs/Configuration.tex

```diff
@@ -1543,16 +1527,16 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
 
   \begin{itemize}
     \tightlist
-    \item CSM region can be marked as boot services code or data, which
+    \item The CSM region can be marked as boot services code, or data, which
       leaves it as free memory for XNU kernel.
-    \item MMIO regions can be marked as reserved memory and stay unmapped,
-      but may be required to be accessible at runtime for NVRAM support.
+    \item MMIO regions can be marked as reserved memory and stay unmapped.
+      They may however be required to be accessible at runtime for NVRAM support.
   \end{itemize}
 
-  This quirk attempts to fix types of these regions, e.g. ACPI NVS for
+  This quirk attempts to fix various types of these regions, e.g. ACPI NVS for
```

Comment:
Is it various? Or is it at least _attempting_ to fix all types? In which case suggest "the types" instead of "various types".

Reply:
Not clear.

cc @vit9696

Reply:
The quirk is fixing memory descriptor types, the change is not correct.